ThumbnailItem/ThumbnailItem.js

```diff
@@ -40,11 +91,16 @@ const ThumbnailItemBase = kind({
 		publicClassNames: true
 	},
 
-	render: ({children, css, src, ...rest}) => {
+	computed: {
+		className: ({type, selected, styler}) => styler.append({
+			roundThumbnail: type === 'round',
+			selected
+		})
+	},
+
+	render: ({css, children, src, ...rest}) => {
 		return (
-			<Item
-				{...rest}
-			>
+			<Item {...rest} css={css}>
 				<slotBefore>
 					<img
 						className={css.thumbnail}
```

Comment:
Let's insert the `slotBefore` prop after this `<img>` tag so someone could still specify a `slotBefore` prop on `ThumbnailItem` and that wouldn't be overridden by this definition.

I think we also need to wrap ThumbnailItem in `Slottable` with the slots defined so that API still works as expected to an app.

Reply:
Added `slotBefore` prop to img. Also, I wrapped the `img` in a ` div` tag to prevent text from shifting while the thumbnail is loading. See more details on Harmony ticket. What do you think of this approach?